samples/shields/x_nucleo_iks01a2/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _x-nucleo-iks01a2-sample:
+
+X-NUCLEO-IKS01A2: MEMS inertial and environmental multi-sensor shield
+#####################################################################
+
+Overview
+********
+This sample enables all sensors of a X-NUCLEO-IKS01A2 shield, and then
+periodically reads and displays data from the shield sensors:
+
+- HTS221: Ambient temperature and relative humidity
+- LPS22HB: Atmospheric pressure and ambient temperature
+- LSM6DSL: 3-Axis Acceleration and 3-Axis Angular Velocity
+- LSM303AGR 3-Axis Acceleration and 3-axis Magnetic field intensity
+
+Requirements
+************
+
+This sample communicates over I2C with the X-NUCLEO-IKS01A2 shield
+stacked on a board with an Arduino connector. The board's I2C must be
+configured for the I2C Arduino connector (both for pin muxing
+and device tree).
```

Comment:
Could you provide a hint (or reference material) that describes how to configure the I2C this way (references to files in this PR or links to external documentation)?

Reply:
OK, I will add reference to the Nucleo F401RE board documentation, which is the one I used
to test the shield sample. The documentation has been updated by @erwango to include the
arduino stuff description and will be merged soon.